NOTICE

```diff
@@ -1,9 +1,2 @@
-Apache Daffodil VS Code Debug
-Copyright 2021 The Apache Software Foundation
-
-This product includes software developed at
-The Apache Software Foundation (http://www.apache.org/).
-
-Based on source code originally developed by
-- VS Code Mock Debug extension, Copyright (c) Microsoft Corporation. All rights reserved. https://github.com/microsoft/vscode-mock-debug
-- DefinitelyTyped version 0.0.1 https://github.com/borisyankov/DefinitelyTyped

```

Comment:
Regarding DefinitelyTyped, it looks like the initial commit to the repo b7b7b2cf75c68c93b32dd592cf315def8ae2b322 included the `typings` directory, which included files that did mention DefinitelyTyped in their headers. The license was added in a later commit. The files were later moved to `src/typings`, but then commit  1a1a9e9dcb43c037482c156593c53e26958f8065 removed all the files in that directory that mentioned DefinitelyTyped in the header in that directory.

Seems to me like this repo doesn't contain any DefiniteltyTyped things anymore and that ThirdPartyLicenes.txt file can be removed.

Reply:
done